respokeSDK/src/main/java/com/digium/respokesdk/RespokeGroup.java

```diff
@@ -93,22 +93,26 @@
         void onError(String errorMessage);
     }
 
-
     /**
      *  The constructor for this class
      *
      *  @param newGroupID  The ID for this group
-     *  @param channel     The signaling channel managing communications with this group 
+     *  @param channel     The signaling channel managing communications with this group
      *  @param newClient   The client to which this group instance belongs
+     *  @param isJoined    Whether the group has already been joined
      */
-    public RespokeGroup(String newGroupID, RespokeSignalingChannel channel, RespokeClient newClient) {
+    public RespokeGroup(String newGroupID, RespokeSignalingChannel channel, RespokeClient newClient,
```

Comment:
Is there anywhere else we expose the RespokeSignalingChannel to the client? (I thought we deliberately didn't.)